v1.0.0.md

```diff
@@ -15,20 +15,35 @@ Harvester can be installed from
 
 # Highlights
 
-* Bring your own SSL certificate [DOC LINK]
-* Air gap environment support [DOC LINK]
-* Proxy support [DOC LINK]
+* Bring your own SSL certificate for the Harvester UI/API. [[DOC](https://docs.harvesterhci.io/v1.0/settings/settings/#ssl-certificates)]
+* Add proxy support to access external services, including the download of images and backup to s3 services. [[DOC](https://docs.harvesterhci.io/v1.0/settings/settings/#http-proxy)]
+* Air gap environment support of both Harvester single cluster and Rancher Harvester node driver support.
+* Allowing Backup and restore VM metadata from the backup target on both existing and new clusters. [[DOC](https://docs.harvesterhci.io/v1.0/vm/backup-restore/)]
+* Harvester RKE1 node driver is now GA and added cloud provider integration. [[DOC](https://docs.harvesterhci.io/v1.0/rancher/rke1-cluster/)]
+* Enhanced Harvester cluster import experience to Rancher. [[DOC](https://docs.harvesterhci.io/v1.0/rancher/virtualization-management/)]
+* Support enable/disable additional data storage partition. [[DOC](https://docs.harvesterhci.io/v1.0/install/harvester-configuration/#installno_data_partition)]
+* Support volume hot-unplugin. [[DOC](https://docs.harvesterhci.io/v1.0/vm/hotplug-volume/)]
+* Multi-disk support enhancement. [[LINKS](https://github.com/harvester/harvester/issues?q=is%3Aissue+label%3Aarea%2Fnode-disk-manager+milestone%3Av1.0.0+)]
+* Support resource over-commit config. [[DOC](https://docs.harvesterhci.io/v1.0/settings/settings/#overcommit-config)]
 
 # Upgrade
 * One click upgrade will be supported from v1.0.0 to future versions.
-* (Experimental) Manual upgrade process is provided to upgrade from v0.3.0 to v1.0.0.
+* (Experimental) Manual upgrade process is provided to upgrade from v0.3.0 to v1.0.0. [[DOC](https://docs.harvesterhci.io/v1.0/upgrade/)]
```

Comment:
Need to update this if we decide to postpone the v0.3 to v1.0 manual upgrade instruction.

Reply:
I think we can still keep it as long as it is marked as experimental, also per Diavd's validation it can be upgraded successfully with only have two known issues(will highlight its workaround steps):
- Failed to access VIP after upgrade, which related to #1645
- Failed to login harvester dashboard by original admin credential, which related to #1666

Reply:
We need to find workarounds for those two issues, otherwise the whole upgrade process is not usable to the end user. That won’t be considered as a successful upgrade.